tools/js-sdk-release-tools/ci.yml

```diff
@@ -52,28 +52,23 @@ extends:
               displayName: 'Install Node.js'
 
             - script: |
-                npm install
-              displayName: 'npm install'
+                npm run install:all
+              displayName: 'npm run install:all'
               workingDirectory: $(System.DefaultWorkingDirectory)/tools/js-sdk-release-tools
             
             - script: |
-                npm run build
-              displayName: 'npm run build'
+                npm run build:all
```

Comment:
https://github.com/Azure/azure-sdk-tools/pull/8751#discussion_r1713198361
@qiaozha  it can be built. the CI has build step